eng/scripts/setup-openssl-env.ps1

```diff
@@ -0,0 +1,36 @@
+# Setup script for OpenSSL environment to enable workspace-wide clippy and builds
+# Run this script before running cargo clippy --workspace or similar commands
+
+Write-Host "Setting up OpenSSL environment for Rust builds..." -ForegroundColor Green
+
+# Set VCPKG environment variables
+$env:VCPKG_ROOT = "C:\vcpkg"
+$env:OPENSSL_DIR = "C:\vcpkg\installed\x64-windows-static-md"
```

Comment:
Why a global directory when we may have different versions for different repos? I have several worktrees (repos) on my machine, which is why the existing docs in our root `CONTRIBUTING.md` say to use a repo-related path.

If you're going to codify those instructions:

1. They should be compatible with what I've already tested works in more ways than this.
2. Those instruction should be updated to have people run this script instead.